VideoAPI/VideoApi.Domain/MeetingRoom.cs

```diff
@@ -6,22 +6,24 @@ internal MeetingRoom()
         {
         }
         
-        public MeetingRoom(string adminUri, string judgeUri, string participantUri, string pexipNode)
+        public MeetingRoom(string adminUri, string judgeUri, string participantUri, string pexipNode, int telephoneConferenceId)
         {
             AdminUri = adminUri;
             JudgeUri = judgeUri;
             ParticipantUri = participantUri;
             PexipNode = pexipNode;
+            TelephoneConferenceId = telephoneConferenceId;
         }
 
         public string AdminUri { get; set; }
         public string JudgeUri { get; set; }
         public string ParticipantUri { get; set; }
         public string PexipNode { get; set; }
+        public int TelephoneConferenceId { get; set; }
 
         public bool IsSet()
         {
-            return AdminUri != null && JudgeUri != null && ParticipantUri != null && PexipNode != null;
+            return AdminUri != null && JudgeUri != null && ParticipantUri != null && PexipNode != null && TelephoneConferenceId > int.MinValue;
```

Comment:
i dont like this bit much :P when are we setting to MinValue? if it can be not "Set" why not make it nullable?